code/modules/projectiles/projectile/energy.dm

```diff
@@ -67,7 +67,7 @@
 /obj/item/projectile/energy/electrode/stunshot
 	nodamage = 0
 	damage = 10
-	agony = 80
+	agony = 50

```

Comment:
More of an opinion but this makes stun revolvers useless. They have a far slower fire rate, lack the ability to pass through objects, take up more space, have slow projectiles, and very few shots to a clip. Add in the fact that with this edit they hardly do more pain than a normal taser and there's no reason to bother with them anymore.